test/pensions/00-all-fields.e2e.spec.js

```diff
@@ -152,6 +152,134 @@ const runTest = E2eHelpers.createE2eTest(
+      .click('.form-panel .usa-button-primary');
+    E2eHelpers.expectNavigateAwayFrom(client, '/financial-disclosure/expectedIncome');
+
+    // other expenses info
+    client.waitForElementVisible('label[for="root_view:hasOtherExpensesYes"]', Timeouts.normal);
+    client.assert.cssClassPresent('.progress-bar-segmented div.progress-segment:nth-child(1)', 'progress-segment-complete');
+    PageHelpers.completeOtherExpensesInfo(client, testData.data);
+    client.axeCheck('.main')
+      .click('.form-panel .usa-button-primary');
+    E2eHelpers.expectNavigateAwayFrom(client, '/financial-disclosure/otherExpenses');
+
+    // spouse net worth info
+    client.waitForElementVisible('label[for="root_spouseNetWorth_bank"]', Timeouts.normal);
+    client.assert.cssClassPresent('.progress-bar-segmented div.progress-segment:nth-child(1)', 'progress-segment-complete');
+    PageHelpers.completeSpouseNetWorthInfo(client, testData.data);
+    client.axeCheck('.main')
+      .click('.form-panel .usa-button-primary');
+    E2eHelpers.expectNavigateAwayFrom(client, '/financial-disclosure/netWorth/spouse');
+
+    // spouse monthly income info
+    client.waitForElementVisible('label[for="root_spouseMonthlyIncome_socialSecurity"]', Timeouts.normal);
+    client.assert.cssClassPresent('.progress-bar-segmented div.progress-segment:nth-child(1)', 'progress-segment-complete');
+    PageHelpers.completeSpouseMonthlyIncomeInfo(client, testData.data);
+    client.axeCheck('.main')
+      .click('.form-panel .usa-button-primary');
+    E2eHelpers.expectNavigateAwayFrom(client, '/financial-disclosure/monthlyIncome/spouse');
+
+    // spouse expected income info
+    client.waitForElementVisible('label[for="root_spouseExpectedIncome_salary"]', Timeouts.normal);
+    client.assert.cssClassPresent('.progress-bar-segmented div.progress-segment:nth-child(1)', 'progress-segment-complete');
+    PageHelpers.completeSpouseExpectedIncomeInfo(client, testData.data);
+    client.axeCheck('.main')
+      .click('.form-panel .usa-button-primary');
+    E2eHelpers.expectNavigateAwayFrom(client, '/financial-disclosure/expectedIncome/spouse');
+
+    // spouse other expenses info
+    client.waitForElementVisible('label[for="root_view:spouseHasOtherExpensesYes"]', Timeouts.normal);
+    client.assert.cssClassPresent('.progress-bar-segmented div.progress-segment:nth-child(1)', 'progress-segment-complete');
+    PageHelpers.completeSpouseOtherExpensesInfo(client, testData.data);
+    client.axeCheck('.main')
+      .click('.form-panel .usa-button-primary');
+    E2eHelpers.expectNavigateAwayFrom(client, '/financial-disclosure/otherExpenses/spouse');
+
+    // first dependent net worth info
+    client.waitForElementVisible('label[for="root_netWorth_bank"]', Timeouts.normal);
+    client.assert.cssClassPresent('.progress-bar-segmented div.progress-segment:nth-child(1)', 'progress-segment-complete');
+    PageHelpers.completeNetWorthInfo(client, testData.data);
```

Comment:
It looks like this is filling these pages in with the data for the applicant, instead of the dependent (my suggestion above would address that).